projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -240,6 +240,15 @@ export function BoostInterstitial() {
 	);
 }
 
+/**
+ * CreatorInterstitial component
+ *
+ * @returns {object} CreatorInterstitial react component.
+ */
+export function CreatorInterstitial() {
+	return <ProductInterstitial slug="creator" />;
```

Comment:
Good catch, yes we should

Reply:
This looks to work already. Just tested with Boost active and then purchased Creator. It landed me in settings with Jetpack active.

Reply:
It doesn't seem to work for me, hmmm 🤔 here are test steps I made:

1. Install Jurassic Ninja with Beta Tester
2. Activate Boost plugin with feature branch `add/creator-to-my-jetpack`
3. Go to "My Jetpack" and I see Creator card there
4. I click "Get Creator", I see interstitial, I proceed to checkout, I'm able to complete purchase
5. I come back to `/wp-admin/admin.php?page=jetpack-beta`
6. But the page is empty - because no Jetpack main plugin is installed

cc @jboland88

Reply:
I've gone ahead and debugged and fixed the issue! The main Jetpack plugin gets installed successfully when JP Creator is activated from JP Boost standalone plugin, with this fix applied: https://github.com/Automattic/jetpack/pull/34307/commits/8b698ac662f0ea2971a5aecd0384ff6794a2313e

Reply:
@robertsreberski - should we add `installsPlugin={true}` here?

Reply:
We do, thanks!